justfile

```diff
@@ -44,6 +44,8 @@ test-pure:
     cd {{frb_pure}}/dart && \
         dart pub get && \
         dart lib/main.dart ../rust/target/debug/{{dylib}}
+test-pure-web:
+    cd {{frb_pure}}/dart && just serve --dart-input lib/main.dart --root web/ -c ../rust --port 8081
```

Comment:
Btw we may also need to test the with_flutter case

(Leave it here to let GitHub prevent accidential merging)